### PR TITLE
min_max

### PR DESCRIPTION
**Pull request by yz398**
Hi Anika,
Do you have any suggestion for me? Thank you so much.

### CODE REVIEW

File: test_min_max_list.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
+# coding: utf-8
```
Comment (anikamukherji) [inline test_min_max_list.py]: I'm not sure where these two comments came from - maybe from when you convert the jupyter notebook to a python file? I would just remove them to avoid confusion

File: test_min_max_list.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# In[15]:
+
+
+import pytest
+
+def test_min_max_list():
+    
+    test_data1 = [0,-3,-1.2,10]
+    test_data2 = [1,3,2,5]
+    test_data3 = [-3/2,-9,-3,-7,-1]
```
Comment (anikamukherji) [inline test_min_max_list.py]: I would test an empty list! So where `test_data4 = []`

File: min_max_list.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+
+# coding: utf-8
+
+# In[37]:
+
+
+def min_max_list(your_list):
```
Comment (anikamukherji) [inline min_max_list.py]: This is a minor thing, but Zhiwei and I made the argument `x`. I think it would be a good idea to keep things consistent and do the same (nothing wrong with what you did - just best to keep the arguments the same). So your function header would read `def min_max_list(x):` and then all references to `your_list` would just be `x`

File: min_max_list.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+
+# coding: utf-8
```
Comment (anikamukherji) [inline min_max_list.py]: Can you remove these two comments as well? Not sure where they came from.

**Comment by yz398** [comment]
Okay. Thank you

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by yz398** [comment]
Hi Anika,
I have changed it into x and delete the first two lines. It is automatically formed by jupyter notebook. If you have any suggestions, let me know. Thank you so much.

File: test_min_max_list.py [hunk 5]
```diff
@@ -1,19 +1,13 @@
-
-# coding: utf-8
-
-# In[15]:
-
-
 import pytest
 
-def test_min_max_list():
+def test_min_max_list(x):
```
Comment (anikamukherji) [inline test_min_max_list.py]: Why did you add an x here? The test function should not take in any parameters

File: test_min_max_list.py (hunk 2 above)
Comment (anikamukherji) [reply test_min_max_list.py]: Can you add this test? ^

File: test_min_max_list.py [hunk 6]
```diff
@@ -1,17 +1,21 @@
...
+    test_data4 = []
     test_answer1 = (-3,10)
     test_answer2 = (1,5)
     test_answer3 = (-9,-1)
+    test_answer4 = (None, None)
     
     assert test_answer1 == min_max_list(test_data1)
     assert test_answer2 == min_max_list(test_data2)
     assert test_answer3 == min_max_list(test_data3)
+    assert test_answer4 == min_max_list(test_data4)
```
Comment (Zhiwei5) [inline test_min_max_list.py]: Great job! I think all your code is in a good shape now.
